Automattic/wp-calypso
# Update Jetpack Restore Stylings

#### Changes proposed in this Pull Request

* Style Restore Flow
* Add restore configuration options

**1. Start restore**
**Mobile**
<img width="300" alt="start restore screenshot mobile" src="https://user-images.githubusercontent.com/2810519/76990175-c559be00-6904-11ea-9a7e-dc693de0905a.png">
**Wide**
<img width="300" alt="start restore screenshot wide" src="https://user-images.githubusercontent.com/2810519/76990166-c0950a00-6904-11ea-95e1-dcf5360db848.png">
**2. Restore in-progress**
**Mobile**
<img width="300" alt="restore in-progress mobile screenshot" src="https://user-images.githubusercontent.com/2810519/76990276-ee7a4e80-6904-11ea-94db-e6be7bb98a7a.png">
**Wide**
<img width="300" alt="restore in-progress wide screenshot" src="https://user-images.githubusercontent.com/2810519/76990389-1ff31a00-6905-11ea-8d2f-ccde2f8f4ac7.png">

**3. Restore finished**
**Mobile**
<img width="300" alt="restore finished mobile screenshot" src="https://user-images.githubusercontent.com/2810519/76990398-22557400-6905-11ea-89d4-dcdf1d35c320.png">
**Wide**
<img width="300" alt="restore finished wide screenshot" src="https://user-images.githubusercontent.com/2810519/76990447-31d4bd00-6905-11ea-84fd-a67bad5d1629.png">

#### Testing instructions



1. Navigate to `/backups/:siteSlug` for a site with backups enables
2. Find a valid backup ( scrolling back in time if necessary ) 
3. Note the date you are backing up to and click "Restore to this point'.
4. Verify you are taken to `/backups/:siteSlug/restore/:rewindId` and it matches the "Start restore" screenshot above. 
5. Verify that you can modify the restore settings.
6. Uncheck at least one restore option, open dev tools and click "Confirm Restore".
7. In the network tab of dev tool find the request `activity-log/:siteId/rewind/to/:rewindId`.
8. inspect the request body/payload and confirm the options matching the checkboxes you setup on step 7. 
9. Verify that the in-progress stage matches the "Restore in-progress" screenshot above.
10. Verify eventually the restore finished and matches the "Restore finished" screenshot above.
11. Verify the "View your website" does indeed takes you to your site.
